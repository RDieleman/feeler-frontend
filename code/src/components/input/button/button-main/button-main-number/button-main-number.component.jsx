import React from "react";
import "./button-main-number.styles.css";
import "../../button.styles.css";

export const ButtonMainNumberComponent = ({content, number = 0, handleOnClick}) =>{
    // Main button
    return(
        <button className="text-body-sec button-main " onClick={() => handleOnClick()}>
            {content}
            <div>{number}</div>
        </button>
    )
}