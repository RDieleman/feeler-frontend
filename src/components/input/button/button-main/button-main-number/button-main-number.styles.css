.button-main > div{
    display: flex;
    justify-content: flex-end;

    color: var(--color-gen-light);

    height: 100%;
    width: auto;;
}