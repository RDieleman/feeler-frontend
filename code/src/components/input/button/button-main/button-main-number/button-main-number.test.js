import React from "react";
import {shallow} from "enzyme";
import {ButtonMainNumberComponent} from "./button-main-number.component";

it("Expect to render ButtonMainNumber component", () => {
    expect(shallow(
        <ButtonMainNumberComponent
            handleOnClick={() => "Button MainNumber clicked"}
            content="test"
            number={10}
        />)).toMatchSnapshot();
});