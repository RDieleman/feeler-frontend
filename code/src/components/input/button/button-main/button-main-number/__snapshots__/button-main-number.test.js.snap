// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render ButtonMainNumber component 1`] = `ShallowWrapper {}`;
